src/main/java/mate/jdbc/dao/CarDao.java

```diff
@@ -0,0 +1,18 @@
+package mate.jdbc.dao;
+
+import java.util.List;
+import mate.jdbc.model.Car;
+
+public interface CarDao {
+    Car create(Car car);
+
+    Car get(Long id);
```

Comment:
on DAO layer method get must return Optional (as well as DriverDao and ManufacturerDao)

Reply:
this is neverending point of arguing about returning Object or Optional in this case.
last time I vice versa was critised for returning Optional instead of Object, because in README was stated - return object.
Let stay as it is.
I think the both cases are acceptable.
or we should clear out it by Bogdan.

Reply:
ok :)